frontend/packages/dev-console/src/components/import/app/ApplicationSelector.tsx

```diff
@@ -49,8 +55,10 @@ const ApplicationSelector: React.FC<ApplicationSelectorProps> = ({
     setApplicationsAvailable(!noApplicationsAvailable);
     availableApplications.current = _.keys(applicationList);
     if (noApplicationsAvailable) {
-      setFieldValue('application.selectedKey', '');
-      setFieldValue('application.name', '');
+      const selField = subPath ? `${subPath}.application.selectedKey` : 'application.selectedKey';
+      const nameField = subPath ? `${subPath}.application.name` : 'application.name';
+      setFieldValue(selField, '');
+      setFieldValue(nameField, '');
```

Comment:
Its good to use `useField` for `application.name` and reference it here (nameField.name). See my previous comment.